src/__test__/pages/experiments/[experimentId]/data-exploration/components/gene-list-tool/ListSelected.test.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { mount, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { Provider } from 'react-redux';
+import { Button, Select } from 'antd';
+import preloadAll from 'jest-next-dynamic';
+import _ from 'lodash';
+import ListSelected from '../../../../../../../pages/experiments/[experimentId]/data-exploration/components/generic-gene-table/ListSelected'
+const TEST_UUID = 'testList';
+const mockStore = configureMockStore([thunk]);
+let component;
+let store;
+const initialState = {
+  genes: {
+    properties: {
+      loading: [],
+      data: {
+        CEMIP: { dispersions: 3.999991789324 },
+        TIMP3: { dispersions: 3.4388329 },
+        SMYD3: { dispersions: 3.1273264798 },
+        I: { dispersions: 0.08756543 },
+        J: { dispersions: 1.352342342 },
+        K: { dispersions: 33.423142314 },
+      },
+      views: {
+        [TEST_UUID]: {
+          fetching: false,
+          error: false,
+          total: 4,
+          data: ['J', 'I', 'K', 'CEMIP'],
+        },
+      },
+    },
+    expression: {
+      loading: [],
+      error: false,
+      data: {},
+    },
+    selected: ['PPBP', 'DOK3', 'YPEL2'],
+    focused: undefined,
+  },
+};
+
+describe('ListSelected', () => {
+  beforeAll(async () => {
+
+    await preloadAll();
+    store = mockStore(initialState);
+
+    component = mount(
```

Comment:
remove this `mount` stuff, since it is not being used (you are currently doing `mount` in each test separately)

Reply:
if you want to do that `mount` stuff for each test automatically, you should put it in `beforeEach` function instead (see how `Embedding.test.jsx` does it for example: https://github.com/biomage-ltd/ui/blob/master/src/__test__/pages/experiments/%5BexperimentId%5D/data-exploration/components/embedding/Embedding.test.jsx#L72)